.dotfiles_meta/Brewfile

```diff
@@ -10,15 +10,16 @@ brew "bash"
 brew "bat"
 brew "blueutil"
 brew "diff-so-fancy"
+brew "exa"
```

Comment:
exa can replace tree via exa --tree
motivation for this change is b/c preview-tui will use tree over exa unless tree is not installed, and tree is pretty ugle